lib/java/com/google/android/material/internal/CollapsingTextHelper.java

```diff
@@ -655,8 +657,16 @@ public void draw(Canvas canvas) {
       }
 
       // Compute where to draw textLayout for this frame
-      final float currentExpandedX =
-          currentDrawX + textLayout.getLineLeft(0) - expandedFirstLineDrawX * 2;
+      final float currentExpandedX;
+      final float currentCollapsedX;
+      if (isRtl) {
+        currentExpandedX = x + textLayout.getLineLeft(0) - expandedFirstLineDrawX * 2 +
+            currentBounds.width() - textLayout.getWidth() * scale;
+        currentCollapsedX = x; //+ currentBounds.width() - collapsedTextWidth * scale;
```

Comment:
Thanks for working on this!

It looks like `currentCollapsedX` is always just `x` in the end like it was before? Also `collapsedTextWidth` isn't used because it's commented out here. But it does look like this does fix the x position of the expanded text.

Reply:
Yep, that was an unfinished try to also correct the position of the collapsed text, but it didn't work out this way.

I currently have a new approach where I try to adjust the `calculateBaseOffsets` function to directly calculate the correct separate x positions for the collapsed and expanded text (in LTR text they are the same, but not in RTL because they are aligned on the right side instead of the left) instead of adjusting them afterwards in `draw()`. Probably that approach is much better, but it's also not working correctly yet... should I push it to another branch so that you can take a look?

Reply:
I'm not sure when I'll be able to take a look to try to help identify the problem based on other priorities that we have. But I still think it would be useful to push what you have for now.

Reply:
Okay, I force-pushed the new approach here and kept the previous one as a backup on our [multiline-collapsingtoolbar-old](https://github.com/material-components/material-components-android/compare/master...opacapp:multiline-collapsingtoolbar-old) branch.

Reply:
Can you rebase the branches on master? We made a change recently which is preventing me from pulling them in. thanks!

Reply:
Okay, should now both be rebased. I accidentally committed the `pseudoLocalesEnabled true` in the build.gradle, that's probably what caused the conflict. Sorry!